users/src/entities/Activity.ts

```diff
@@ -0,0 +1,13 @@
+import { Entity, PrimaryKey, Property } from "@mikro-orm/core";
+
+@Entity({ tableName: "activity" })
+export class Activity {
+    @PrimaryKey()
+    uid!: string;
```

Comment:
(only a nit) I personally prefer `userId` to `uid` (mistaken for unique id)

Reply:
The firebase auth token class calls uid. I just wanted to standardise.